src/main/java/org/flowsoft/flowg/visitors/CodeGeneratingVisitor.java

```diff
@@ -182,11 +187,56 @@ public ExpressionValue Visit(StatementListNode statementListNode) throws Excepti
     public ExpressionValue Visit(MoveNode moveNode) throws Exception {
         var parameters = moveNode.GetChild().GetChildren();
         var point = parameters.get(0).Accept(this).GetPoint();
+        _currentPosition = point;
 
         _stringBuilder
-                .append("G1")
+                .append("G0")
                 // Don't extrude anything we are moving :)
-                .append(" E0")
+                .append(" X").append(point.GetX())
+                .append(" Y").append(point.GetY())
+                .append(" Z").append(point.GetZ())
+                .append('\n');
+
+        return new ExpressionValue(Type.Void);
+    }
+
+    @Override
+    public ExpressionValue Visit(LineNode lineNode) throws Exception {
+        var parameters = lineNode.GetChild().GetChildren();
+        var point = parameters.get(0).Accept(this).GetPoint();
+
+        // All distances in mm
+        var distance = point.Distance(_currentPosition);
+        var filamentDiameter = new BigDecimal("2.85");
+        var nozzleDiameter = new BigDecimal("0.4");
+        var layerHeight = new BigDecimal("0.1");
```

Comment:
Hardcoded values - Need to change at some point

Reply:
Det bliver lavet om når vi har includes og gcode functions :)